apps/token-manager/app/src/screens/EmptyState.js

```diff
@@ -7,9 +7,9 @@ const EmptyState = ({ onActivate }) => (
   <Main>
     <EmptyStateCard
       icon={<img src={emptyIcon} alt="" />}
-      title="There are no tokens assigned yet"
-      text="Assign tokens to get started."
-      actionText="Assign Tokens"
+      title="There are no tokens added yet"
+      text="Add tokens to get started."
+      actionText="Add Tokens"
```

Comment:
@dizzypaty I wonder if we could use some language like

"There are no token holders yet"

"Add tokens to an address to get started"

Reply:
Yes, that sounds better!